React/Views/RCTModalHostViewManager.m

```diff
@@ -104,7 +104,7 @@ - (void)invalidate
   for (RCTModalHostView *hostView in _hostViews) {
     [hostView invalidate];
   }
-  [_hostViews removeAllObjects];
+  _hostViews = [NSPointerArray weakObjectsPointerArray];
```

Comment:
Can it be just `_hostViews = nil;`?

Reply:
Currently, after invalidation, module should be deallocated, seems we can just nil it. :)